apps/evm/src/pages/Market/Market/MarketHistory/index.tsx

```diff
@@ -13,8 +15,23 @@ export const MarketHistory: React.FC<MarketHistoryProps> = ({
   asset,
   poolComptrollerContractAddress,
 }) => {
-  const { data: chartData, isLoading: isChartDataLoading } = useGetChartData({
+  const [selectedSupplyPeriod, setSelectedSupplyPeriod] = useState<MarketHistoryPeriodType>('year');
+  const [selectedBorrowPeriod, setSelectedBorrowPeriod] = useState<MarketHistoryPeriodType>('year');
```

Comment:
UX-wise, do you think it would make sense to share the selected period between both graphs? I Imagine a user would want to compare them side by side.

Reply:
I'm fine with both ways, one gives you independent controls and the other saves you an extra click. Let's try sharing the periods and see if everyone likes it.

Another thing here is that I should probably restore having 30d as the default option.

Reply:
Sounds good to me.

Yeah, 30 days might be better for most use cases.